ftva_lab_data/views.py

```diff
@@ -1,3 +1,75 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from .forms import ItemForm
+from .models import SheetImport
 
-# Create your views here.
+
+def add_item(request):
+    if request.method == "POST":
+        # save a new SheetImport object
+        form = ItemForm(request.POST)
+        if form.is_valid():
+            form.save()
+            messages.success(request, "Item added successfully!")
+            return redirect("view_item", item_id=form.instance.id)
+
+        else:
+            # Handle form errors, if needed
+            messages.error(request, "Please correct the errors below.")
+            return render(
+                request,
+                "add_edit_item.html",
+                {"form": form, "title": "Add Item", "button_text": "Add Item"},
```

Comment:
Since the same context (the same collection of key/value pairs) is passed in multiple cases (here and line 30), please define a `context` dictionary and reference it in both places.

Reply:
Context dictionaries added to the `add_item` and `edit_item` views. Ready for re-review!